code/game/objects/structures/crates_lockers/closets.dm

```diff
@@ -7,6 +7,8 @@
 	layer = BELOW_OBJ_LAYER
 	w_class = ITEM_SIZE_GARGANTUAN
 	matter = list(MATERIAL_STEEL = 10)
+	bad_types = "/obj/structure/closet"
```

Comment:
Why this line is enclosed in double quotes ?

Reply:
for many reasons but none specific. The code also accepts it in this format.

Reply:
Okay